frontends/web/src/app.jsx

```diff
@@ -137,10 +141,12 @@ export class App extends Component {
                         <Settings
                             path="/settings"
                             deviceIDs={deviceIDs} />
+                        {/* Use with TypeScript: {Route<{ deviceID: string }>({ path: '/manage-backups/:deviceID', component: ManageBackups })} */}
                         <ManageBackups
                             path="/manage-backups/:deviceID"
-                            showCreate={true}
-                            deviceIDs={deviceIDs} />
+                            // showCreate={true} // Does not exist!
+                            // deviceIDs={deviceIDs} // Does not exist!
```

Comment:
Maybe just remove these? If you do it in a separate commit on top of the existing work, then it can still be traced back in git history, i.e. you'd see first commented out with "// does not exist" and then removed completely in a next commit.

Reply:
Yes, thought about this as well but especially the `showCreate` property that doesn't exist on the other route seems like a bug that might need proper fixing (and its own issue, which I still have to create). Plus, I wanted to refactor app.jsx to app.tsx anyway because TypeScript struggles with the implicit insertion of `deviceID` by the router. This is the reason why I decided to opt out completely of TS checking for the entire file: https://github.com/digitalbitbox/bitbox-wallet-app/pull/71/files/29014f71f7a2849bd07c8f3e5c05f1b41863e74c#diff-bc69595256b9d37e9f0a9653514577c7R17
I couldn't find out how to specify the generic type for TS through JSDoc comments (as I could do in other places) and just ignoring the following line with `@ts-ignore` didn't work for some reason… 🙁

Reply:
I just saw that `ManageBackups` has the `showCreate={true}` on its child and I will remove those lines in another PR then.
https://github.com/digitalbitbox/bitbox-wallet-app/blob/a85484ae01ed4dc8295915bbb3c5a311048fd4da/frontends/web/src/routes/device/manage-backups/manage-backups.jsx#L36-L45